pages/How-to/guide/assigning-issue.md

```diff
@@ -1,4 +1,60 @@
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
+
+
+
+<div>
+  <h1>Creating and assigning an issue from a repository</h1>
+  
+  <p> Issues are important as they alert a maintainer to review and make new changes or receive information. These Information could include bugs, new ideas, feedback etc.
```

Comment:
There are some spelling mistakes which you can take care of like "Information" should be "information".

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Unreadable